.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+<!-- Thank you for trying to improve Rust through the RFC process! -->
+<!-- Please add a short summary of your RFC below and add the link to the markdown file to the "Rendered" link below -->
```

Comment:
The rendered link is automated by a bot. I don't think we need to bother authors with it.

```suggestion
<!-- Please add a short summary of your RFC below -->
```